### PR TITLE
Handle decoding of actual AEP check content

### PR DESCRIPTION
**Pull request by itzg**

### CODE REVIEW

File: poller/session_test.go [hunk 1]
```diff
@@ -280,12 +280,12 @@ func TestEleSession_HandshakeTimeoutStoppedOnSuccess(t *testing.T) {
...
-		      /   /      end of test
-		     /   /      /       first heartbeat would have been sent
-	      	/   /      /       /
-		  0    10     20      30   ms
+			        soon after this the handshake response is "sent back"
+			       /   auth timeout would have fired here, but should be stopped
+			      /   /      end of test
+			     /   /      /       first heartbeat would have been sent
+		      	/   /      /       /
+			  0    10     20      30   ms
```
Comment (dimtruck) [inline poller/session_test.go]: Nit: just an extra tab?

File: protocol/messages_test.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestDecodePollerPrepareBlockRequest(t *testing.T) {
+
+	file, err := os.Open("testdata/poller-prepare-block.json")
+	require.NoError(t, err)
```
Comment (dimtruck) [inline protocol/messages_test.go]: Nice, never seen that before

File: poller/session_test.go (hunk 1 above)
Comment (itzg) [inline poller/session_test.go]: Yeah, `go fmt` or IntelliJ is messing with me there.

File: protocol/messages_test.go (hunk 2 above)
Comment (itzg) [inline protocol/messages_test.go]: Yep, it's super handy

**Pull request merged**